From 03f4203e2ceb5b51dd7663d1ee7ed68042f563d0 Mon Sep 17 00:00:00 2001
From: Ryan Kavanagh <rak@debian.org>
Date: Fri, 11 Dec 2020 15:54:17 -0500
Subject: [PATCH] Drop priority field from docx2txt.mime (Closes: #977157)

This field does not appear in the standard [RFC1524] and causes
mailcap-parsing programs (e.g., s-nail) to complain about ill-formed
mailcap entries.
---
 debian/docx2txt.mime | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/docx2txt.mime b/debian/docx2txt.mime
index c4ee279..1a486de 100644
--- a/debian/docx2txt.mime
+++ b/debian/docx2txt.mime
@@ -1 +1 @@
-application/vnd.openxmlformats-officedocument.wordprocessingml.document; docx2txt '%s' - ; copiousoutput; description=Office Open XML Document; prority=1
+application/vnd.openxmlformats-officedocument.wordprocessingml.document; docx2txt '%s' - ; copiousoutput; description=Office Open XML Document
-- 
2.29.2

